Yoast/wordpress-seo
# Update changelog at subsequent RC build

## Summary


This PR can be summarized in the following changelog entry:

* [not-user-facing]

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* See steps in the issue, additionally run the following command: `grunt update-readme --plugin-version=12.7`
* Your defined editor should pop-up with the entire changelog file
* The 12.6 entries should also be removed from the changelog (not the 12.7 ones)

## UI changes

## Documentation

## Quality assurance


Fixes #14098
